stablehlo/dialect/VhloOps.cpp

```diff
@@ -333,6 +333,20 @@ LogicalResult verifyConstraint_0_17_0(mlir::Operation* op,
     return failure();
   return success();
 }
+
+LogicalResult verifyConstraint_1_2_1(mlir::Operation* op,
```

Comment:
All opset changes require a minor version bump. This should be 1.3.0. A few changes needed for this:

- [ ] Update Version.h to 1.3.0 since this is an opset change
- [ ] Update [VhloDialect.td](https://github.com/openxla/stablehlo/blob/8a78ebb89bc6c5608f6f27a7a9ce687b3578b79d/stablehlo/dialect/VhloDialect.td#L42) with the opset change.
- [ ] Rename this method to `_1_3_0`